components/service/nimbus/src/main/java/mozilla/components/service/nimbus/Nimbus.kt

```diff
@@ -180,8 +180,10 @@ private val loggingErrorReporter: ErrorReporter = { e: Throwable ->
 /**
  * A implementation of the [NimbusApi] interface backed by the Nimbus SDK.
  */
-@Suppress("LargeClass")
+@Suppress("LargeClass", "LongParameterList")
 class Nimbus(
+    private val appName: String,
+    private val channel: String,
```

Comment:
By convention,`Context` is the first argument.

Also: I think I'd suggest putting the channel and app name in a small `data class` object, `NimbusAppSettings`.